### PR TITLE
Update installation guide

### PR DESCRIPTION
**Pull request by ilonashub**

### CODE REVIEW

File: _posts/2012-04-30-commenting.markdown [hunk 1]
```diff
@@ -74,9 +45,9 @@ class Comment < ActiveRecord::Base
 belongs_to :idea
 {% endhighlight %}
 
-## שלב 5: הדפיסי למסך את טופס הוספת התגובה ואת רשימת התגובות הקיימות
+## *3.* הדפיסי למסך את טופס הוספת התגובה ואת רשימת התגובות הקיימות
 
-פתחי את app/views/ideas/show.html ואחרי ה-image_tag
+Open app/views/ideas/show.html.erb and after the image_tag
```
Comment (rotev) [inline _posts/2012-04-30-commenting.markdown]: Need to translate this line :)

File: _posts/2012-04-30-commenting.markdown [hunk 2]
```diff
@@ -84,27 +55,29 @@ belongs_to :idea
...
     <br />
     <p><%= comment.body %></p>
+    <p><%= link_to 'Delete', comment_path(comment), method: :delete, data: { confirm: 'Are you sure?' } %></p>
   </div>
 <% end %>
 <%= render 'comments/form' %>
 {% endhighlight %}
 
-בקובץ app/controllers/ideas_controller.rb הוסיפי בפעולת ה-show אחרי השורה
+In `app/controllers/ideas_controller.rb` add to show action after the row
```
Comment (rotev) [inline _posts/2012-04-30-commenting.markdown]: ... and this one too.

File: _posts/2012-04-30-commenting.markdown [hunk 3]
```diff
@@ -84,27 +55,29 @@ belongs_to :idea
...
 {% endhighlight %}
 
 את
 {% highlight ruby %}
+@comments = @idea.comments.all
 @comment = @idea.comments.build
 {% endhighlight %}
 
-פתחי את הקובץ app/views/comments/_form.html ולאחר
+Open `app/views/comments/_form.html.erb` and after
```
Comment (rotev) [inline _posts/2012-04-30-commenting.markdown]: and this one as well.

File: _posts/2012-04-30-commenting.markdown [hunk 4]
```diff
@@ -117,4 +90,12 @@ belongs_to :idea
 <%= f.hidden_field :idea_id %>
 {% endhighlight %}
 
+next, remove
```
Comment (rotev) [inline _posts/2012-04-30-commenting.markdown]: oh, and this should also be translated.

File: _posts/2012-04-30-commenting.markdown [hunk 5]
```diff
@@ -84,27 +55,29 @@ belongs_to :idea
 הוסיפי
 {% highlight erb %}
 <h3>Comments</h3>
-<% @idea.comments.each do |comment| %>
+<% @comments.each do |comment| %>
   <div>
     <strong><%= comment.user_name %></strong>
     <br />
```
Comment (rotev) [inline _posts/2012-04-30-commenting.markdown]: I'd consider removing the XHTML style `<br />`'s (and just use `<br>` instead).

File: _posts/2012-04-19-heroku.markdown [hunk 6]
```diff
@@ -93,20 +103,22 @@ Git remote heroku added
...
-Compressing objects: 100% (115/115), done.
-Writing objects: 100% (134/134), 35.29 KiB, done.
-Total 134 (delta 26), reused 0 (delta 0)
-
------> Heroku receiving push
------> Ruby/Rails app detected
------> Installing dependencies using Bundler version 1.1.2
-       Running: bundle install --without development:test --path vendor/bundle --binstubs bin/ --deployment
-       Fetching gem metadata from https://rubygems.org/.......
+Initializing repository, done.
```
Comment (rotev) [inline _posts/2012-04-19-heroku.markdown]: This guide now contains a lot of untranslated text.

**Comment by ilonashub** [comment]
@rotev updated the site according to your notes. Thanks!

**Pull request merged**